test/ComponentTests/TestAssets/TestProjectForAssemblyResolution/TestProjectForAssemblyResolution.csproj

```diff
@@ -7,9 +8,4 @@
     <AssemblyOriginatorKeyFile>$(RepoRoot)scripts\build\key.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="$(Legacy__MicrosoftNETTestSdkVersion)" />
```

Comment:
We actually don't care if that's a test project, all we care about is that the project type is loaded in memory because specified in the runsettings. Removing this reference avoid Test Explorer to discover and load the dlls in memory, causing side effects to our integration test.